test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -112,7 +114,15 @@ public static void Stress(int concurrency = 0)
                 });
             });
 
+        watchForTotal.Stop();
+        var cntLoopsTotal = (ulong)statistics.Sum();
+        var totalLoopsPerSecond = (double)cntLoopsTotal / ((double)watchForTotal.ElapsedMilliseconds / 1000.0);
+        var cntCpuCyclesTotal = GetCpuCycles();
+        var cpuCyclesPerLoopTotal = cntLoopsTotal == 0 ? 0 : cntCpuCyclesTotal / cntLoopsTotal;
         Console.WriteLine(output);
+        Console.WriteLine("Stats from entire duration:");
+        var totalOutput = $"Loops: {cntLoopsTotal:n0}, Loops/Second: {totalLoopsPerSecond:n0}, CPU Cycles/Loop: {cpuCyclesPerLoopTotal:n0}";
```

Comment:
This could be confusing since it's already taken previously (and been documented).
Consider taking a different approach:

```text
Stress test stopped by the user:

* Total Loops: 100,000,000
* Average Loops per Second: 100,000
* Average CPU Cycles per Loop: 785
```